README.md

```diff
@@ -1090,6 +1108,14 @@ mySwitch.toggle()
 aView.convertLocalizables()
 ```
 
+**Change the frame of the view easily**
+```swift
+aView.x += 100 // move  to right
+aView.y += 100 // move downwards
+aView.width -= 10 // make the view narrower
+aView.height -= 10 // make the view shorter 
+```
+
 It will iterate on all the subviews of the view, and use the text / placeholder as key in `NSLocalizedString`.
```

Comment:
That comment must be below `Automates your localizables` and above the new section.